rwengine/src/audio/SfxParameters.cpp

```diff
@@ -0,0 +1,177 @@
+    {33, 404, 80},
+    {34, 405, 30},
+    {35, 405, 30},
+    {36, 407 /*-408*/, 30},   /// 407 plays continuously
+                              ///while 408 plays intermittently
+    {37, 407 /*-408*/, 30},   /// Liberty City
+                              /// Sawmills/Bitch'N' Dog Food
+    {38, 409, 30},
+    {39, 409, 80},
+    {40, 410 /*-411*/, 30},   /// Both plays continuously
+    {41, 410 /*-411*/, 30},   /// Mr. Wong's Laundrette
+    {42, 412, 30},            /// Roast Peking Duck
+    {43, 412, 80},
+    {44, 413, 30},            /// Cipriani's Ristorante
+    {45, 413, 80},
+    {46, 414, 30},
+    {47, 414, 30},
+    {48, 415, 30},            /// Marco's Bistro (no sound)
+    {49, 415, 80},
+    {50, 416 /*-419*/, 30},   /// Plays separately and intermittently
+    {51, 416 /*-419*/, 80},   /// Francis International Terminal B
+    {52, 420 /*-422*/, 30},
+    {53, 420 /*-422*/, 30},   /// Chinatown (no sound)
+    {54, 423 /*-425*/, 30},   /// Plays separately and intermittently
+    {55, 423 /*-425*/, 80},
+    {56, 426, 30},
+    {57, 426, 80},            /// Portland Docks (no sound)
+    {58, 427 /*-431*/, 30},   /// Plays separately and intermittently
+    {59, 427 /*-431*/, 80},   /// Left side of Misty's apartment
+    {60, 406, 30},            /// Salvatore's place
+    {61, 406, 80},
+    {62, 432 /*-434*/, 20},   /// South of Sex Club Seven
+                              /// 432 plays continuously
+                              /// while 433-434
+                              /// plays separately
+                              /// and intermittently
+    {63, 432 /*-434*/, 80},
+    {64, 435 /*-437*/, 20},   /// East of Portland
+                              /// Pay 'N' Spray 435 plays
+                              /// continuously while
+                              /// 436-437 plays separately
+                              /// and intermittently
+    {65, 435 /*-437*/, 80},
+    {66, 438 /*-440*/, 20},  /// Executive Relief 438
+                             /// plays continuously while
+                             /// 439-440 plays separately
+                             /// and intermittently
+    {67, 438 /*-440*/, 80},
+    {68, 442, 30},
+    {69, 442, 80},           /// Bank of Liberty/Staunton
+                             /// police HQ alarm
+    {70, 441, 30},
+    {71, 441, 80},           /// Old school hall
+    {72, 443, 30},
+    {73, 443, 80},           /// Warehouse rave
+    {76, 179 /*-184*/, 30},  /// Plays separately and intermittently
+    {77, 179 /*-184*/, 80},  /// Staunton police HQ
+    {84, 444, 30},
+    {85, 444, 80},
+    {86, 444, 30},
+    {87, 444, 80},
+    {88, 445, 30},
+    {89, 445, 80},
+    {90, 433 /*-434*/, 20},  /// Plays separately and intermittently
+    {91, 433 /*-434*/, 80},  /// Right side of Misty's apartment
+    {102, 157, 50}           /// Callahan Bridge fire
+};
+}  // namespace
+
+int convertScriptIndexIntoSfx(int scriptId) {
```

Comment:
Does ``SoundInstanceData`` need to be hidden? A simpler interface could just be ``const SoundInstanceData& getScriptSoundInstanceData(int scriptId)``.

Reply:
Not sure if we need/want this interface in script. If you want, I can change.

Reply:
Yeah. This seems to be unused in this PR, could you move it into the next PR?